.github/workflows/marketing-app-deploy.yml

```diff
@@ -159,6 +159,14 @@ jobs:
         parameter-overrides: "ContainerImageHashDigest=${{ needs.build-and-push-image.outputs.digest }},BaseHostedZoneId=${{ vars.MARKETING_BASE_HOSTED_ZONE_ID }},BaseDomainName=${{ vars.MARKETING_BASE_DOMAIN }},SubdomainName=${{ vars.MARKETING_SUBDOMAIN_NAME }},CloudFrontTLSCertificateArn=${{ vars.MARKETING_CLOUDFRONT_TLS_CERTIFICATE_ARN }}"
         capabilities: CAPABILITY_IAM, CAPABILITY_NAMED_IAM
 
+    - name: "[Test] Invalidate CloudFront Cache"
+      env:
+        CLOUDFRONT_DISTRIBUTION_ID: ${{ vars.CLOUDFRONT_DISTRIBIUTION_ID }}
```

Comment:
```suggestion
        CLOUDFRONT_DISTRIBUTION_ID: ${{ vars. CLOUDFRONT_DISTRIBUTION_ID }}
```

Reply:
Just wanted to confirm - I only see the addition of a space here which wouldn't be valid. Did you have a different review comment?

Reply:
oops. sorry about that. fixed the missing space.

Reply:
The spelling of distribution is incorrect